src/ReleaseHistory.md

```diff
@@ -410,4 +410,5 @@
 * Change schemas back to draft-04 to reenable Intellisense in the Visual Studio JSON editor.
 
 ## **v2.1.6** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.6) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.6) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.6) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.6)
-* BUGFIX: Fortify FPR converter does not populate originalUriBaseIds if the source is a drive letter (e.g. C:)
\ No newline at end of file
+* BUGFIX: Fortify FPR converter does not populate originalUriBaseIds if the source is a drive letter (e.g. C:)
+* BUGFIX: Multitool rebaseUri command throws null reference exception if results reference run.artifacts using indexes.
```

Comment:
>indexes [](start = 111, length = 7)

"indexes" => "indices" if you want the English plural, or "the index property" (my preference).